steps.md

```diff
@@ -1,12 +1,16 @@
 # install depdendencies
-docker-compose exec php-app composer install
+docker compose exec php-app composer install
 
 # create migrations
-docker-compose exec php-app bin/cake bake migration NameOfMigration
+docker compose exec php-app bin/cake bake migration NameOfMigration
 
 # run migrations
-docker-compose exec php-app bin/cake migrations migrate
+docker compose exec php-app bin/cake migrations migrate
 
 
 # create controller
-bin/cake bake controller Name
\ No newline at end of file
+docker compose exec php-app bin/cake bake controller Name
+
+
+# install authtentication plugin
+docker compose exec php-app composer require "cakephp/authentication:^2.4"
```

Comment:
```suggestion
docker compose exec php-app composer require cakephp/authentication
```

If I'm not mistaken that we can remove the version. Therefore, the composer will find the latest version compatible with the current CakePHP version. Please check.

Reply:
it works, installs if there is not installed it otherwise updates, in my case there was no changes